Feature/blog series list fix

daquinohd opened the pull request.

daquinohd commented:
This replaces the previous PR for the Blog Listing View stories (#1017) 
*ODE:* http://ncigovcdode50.prod.acquia-sites.com/

@VictoriaSunNIH - this addresses the feedback from #1017, minus this issue:
> For Spanish, the month is appearing correctly, but the format of the date is not. In Spanish, the day should appear first — 11 de marzo de 2019 — but what I'm saying on the ODE is "Marzo 11, 2019"

VictoriaSunNIH commented:
@lburack  - this is looking good to me.  Please test when possible (as its becoming a blocker for @daquinohd )

lburack commented:
Looking good to me, too! I did log a bug for changing the number of posts to display on the blog series not working as expected (#1057). But otherwise this passes IA review!

daquinohd commented:
@bryanpizzillo - here's the squashed commit message: 
```
(Closes #975) 
* Update blog series template
* Remove intro_text and body fields
* Add public_use (will work when #860 is merged to develop)
* Fixed yaml_content

(Closes #827)
* Renamed page_manager variants
* Added layouts and configurations for page manager
* Translation & layout plugin name change
* Set HTML column structure for blog post & series
* Added translation for 'continue reading' and 'by'
* Configured thumbnail crop for Blog Series list
* Configured date format for all blog content
* Added blog_macros.twig

(Closes #994)
* Made short title mandatory 

Bug fixes:
* Updated blog series display mode configs
* Rolled back settings config
* Added translation blt custom job
* Removed extra whitespace from byline
```

bryanpizzillo commented on docroot/profiles/custom/cgov_site/modules/custom/cgov_blog/src/Services/BlogManager.php:
```diff
@@ -0,0 +1,10 @@
+<?php
```
If you are not using the manager can you please remove it?

daquinohd replied:
Manager class removed.

bryanpizzillo commented on docroot/profiles/custom/cgov_site/modules/custom/cgov_blog/src/Plugin/Block/CgovBlogFeaturedPosts.php:
```diff
@@ -0,0 +1,82 @@
+  public function build() {
+    $build = [
+      '#markup' => $this->t('
+        <div class="contentid-1070871 slot-item first-SI">
```
Uhhh.... Can you either implement this or not include it?  If you are doing this so that it may be styled, then please create a feature branch for blogs.

daquinohd replied:
CgovBlogFeaturedPlugin removed.
